testing/fixtures/when.go

```diff
@@ -132,8 +132,8 @@ func (w *When) DeletePipelineAndWait(timeoutArgs ...time.Duration) *When {
 		w.t.Fatal(err)
 	}
 	timeout := defaultTimeout
-	if len(timeoutArgs) > 0 {
-		timeout = timeoutArgs[0]
+	if len(argTimeout) > 0 {
```

Comment:
why do we need an array instead of a single Duration argument?

Reply:
because if we are making it variadic function ,we can pass 0 or more arguments ,in this way existing usage wont be affected if any  in other libraries @KeranYang